backend/models/question.py

```diff
@@ -4,6 +4,28 @@
 
 from backend.constants import QUESTION_TYPES, REQUIRED_QUESTION_TYPE_DATA
 
+_TESTS_TYPE = t.Union[t.Dict[str, str], int]
+
+
+class Unittests(BaseModel):
+    """Schema model for unittest suites in code questions."""
+    allow_failure: bool = False
+    tests: _TESTS_TYPE
+
+    @validator("tests")
+    def validate_tests(cls, value: _TESTS_TYPE) -> _TESTS_TYPE:
+        """Confirm that at least one test exists in a test suite."""
+        if isinstance(value, dict) and not len(value.keys()):
```

Comment:
```suggestion
        if isinstance(value, dict) and len(value.keys() == 0):
```

This line took me longer than I care to admit to comprehend, something more explicit like this could be useful. Going to approve anyway, since it might just be me.

Reply:
`not value` could work as well